analyzer/src/test/kotlin/managers/utils/NuGetSupportTest.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers.utils
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.containExactly
+import io.kotest.matchers.should
+
+import java.io.File
+
+class NuGetSupportTest : WordSpec() {
+    init {
```

Comment:
Please pass `init` as a lambda to the `WordSpec` contrsuctor to reduce indentation, like at

https://github.com/oss-review-toolkit/ort/blob/a879f6f865c0618513c54a39f961baf73556d763/analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt#L31